lib/gui/components/svg-icon.js

```diff
@@ -53,9 +54,24 @@ class SVGIcon extends react.Component {
     // relatively to the `index.html`.
     const imagePath = path.join(__dirname, this.props.path);
 
-    const contents = fs.readFileSync(imagePath, {
-      encoding: 'utf8'
-    });
+    let contents;
```

Comment:
The linter's yelling about this not being initialized to a value; maybe just set it to an empty string; `let contents = '';`